ludwig/automl/auto_tune_config.py

```diff
@@ -1,20 +1,23 @@
 import copy
+import logging
 from collections import OrderedDict
 
 import psutil
-import ray
 
 try:
     import GPUtil
 except ImportError:
     raise ImportError(" ray is not installed. " "In order to use auto_train please run " "pip install ludwig[ray]")
 
 from ludwig.api import LudwigModel
-from ludwig.automl.utils import get_available_resources, get_model_name
+from ludwig.automl.utils import get_model_name
 from ludwig.constants import BATCH_SIZE, HYPEROPT, PREPROCESSING, SPACE, TRAINER
 from ludwig.data.preprocessing import preprocess_for_training
 from ludwig.features.feature_registries import update_config_with_metadata
 from ludwig.utils.defaults import merge_with_defaults
+from ludwig.utils.torch_utils import initialize_pytorch
+
+logger = logging.getLogger(__name__)
```

Comment:
I may be wrong here -- I believe with https://github.com/ludwig-ai/ludwig/pull/1669, we don't need to set up extra logging objects in torch -- `logging.info()` should just work.

Reply:
Yes, that works, thanks.  Done.